### PR TITLE
Main+platform on trigger

### PR DESCRIPTION
**Pull request by KishoreKB2711**
Final Demo Main + PushUp and MoveTo Features

### CODE REVIEW

File: Atom/src/core/GraphicsManager.cpp [hunk 1]
```diff
@@ -45,7 +45,8 @@ void GraphicsManager::init() {
 	mode = glfwGetVideoMode(monitor);
 
 	// Create a windowed mode window and its OpenGL context
-	mWindow = glfwCreateWindow(width, height, title.c_str(), monitor, nullptr);
+	//mWindow = glfwCreateWindow(width, height, title.c_str(), monitor, nullptr);
```
Comment (auxeon) [inline Atom/src/core/GraphicsManager.cpp]: undo these lines

**Comment by KishoreKB2711** [comment]
No Levels yet

File: imgui.ini [hunk 2]
```diff
@@ -5,7 +5,7 @@ Collapsed=0
 
 [Window][OVERLAY]
 Pos=60,60
-Size=188,46
+Size=181,46
```
Comment (auxeon) [inline imgui.ini]: undo this file as well

**Review by auxeon — COMMENTED** [review]
fix these

**Pull request merged**